### PR TITLE
Bugfix: Fixed edition issue with rustfmt.

### PR DESCRIPTION
**Pull request by Uzaaft**
# Description

This PR attempts to solve an issue with rustfmt not working. Tested on my own machine(MacBook Pro) on a rust crate created with `cargo init --bin`, 
I think it's a sensible assumption to assume that people will write code for the 2018 edition of rust.

Example output when running on a repository with async: 
image

## How Has This Been Tested?

- Created a repo with: `cargo init --bin`
- Indented a line too much.
- Run command `:FormatWrite` or `:w`

### CODE REVIEW

File: lua/default-config.lua [hunk 1]
```diff
@@ -223,7 +223,7 @@ O = {
       -- @usage can be clippy
       formatter = {
         exe = "rustfmt",
-        args = { "--emit=stdout" },
+        args = { "--emit=stdout --edition=2018" },
```
Comment (abzcoding) [inline lua/default-config.lua:226]: can you make it into an actual array like
```lua
args = { "--emit=stdout", "--edition=2018" },
```

**Comment by ChristianChiarulli** [comment]
also we need to run that silent in whichkey or a pcall

File: lua/default-config.lua (hunk 1 above)
Comment (Uzaaft) [reply lua/default-config.lua]: Done.

**Comment by abzcoding** [comment]
@Uzaaft while at it, can you change this line inside `lua/lv-which-key/config.lua` 🥺 
```lua
      f = { "<cmd>FormatWrite<cr>", "Format" },
```
to
```lua
      f = { "<cmd>silent FormatWrite<cr>", "Format" },
```

**Comment by Uzaaft** [comment]
@ChristianChiarulli  That's sounded all greek to me. Do you have a guide for that? Or somewhere I can look to begin understanding what you just said... 😅

**Comment by abzcoding** [comment]
> @ChristianChiarulli That's sounded all greek to me. Do you have a guide for that? Or somewhere I can look to begin understanding what you just said... 😅

he meant the thing that i wrote

**Comment by Uzaaft** [comment]
Ah sweet. Fixing it now

**Pull request merged**